extensions/amp-story/1.0/amp-story.js

```diff
@@ -2783,7 +2783,8 @@ export class AmpStory extends AMP.BaseElement {
    */
   static isBrowserSupported(win) {
     return Boolean(
-      win.CSS && win.CSS.supports && win.CSS.supports('display', 'grid')
+      win.CSS && win.CSS.supports && win.CSS.supports('display', 'grid') &&
+      win.CSS.supports('color', 'var(--test)')
```

Comment:
The linter is now very opinionated:

```suggestion
      win.CSS &&
        win.CSS.supports &&
        win.CSS.supports('display', 'grid') &&
        win.CSS.supports('color', 'var(--test)')
```

Reply:
And the GitHub edit UI can't keep up 😦